### ISSUE TITLE
A section on Configuration

### ISSUE THREAD

**Issue by k4ml** [orchestral/docs#8]
I suggest a specific section on configuration options available. As a beginner (both to orchestral and also laravel), I have to dig through the source code of `vendor/orchestra/foundation/src/config/config.php` and `vendor/orchestra/foundation/src/Orchestra/Foundation/Abstractable/RouteManager.php` to figure out how to change the default path from '/admin' to something else and then make the configuration in my app instead of in the vendor package.

Basically the section should list out all configurable options available, how to use the artisan command to generate app config and a link to laravel docs on [package config][1] if user want to learn more on the configuration. I can work on the patch.

[1]:http://laravel.com/docs/packages#package-configuration
